.github/workflows/vast.yaml

```diff
@@ -299,7 +299,7 @@ jobs:
           cmake -B "$BUILD_DIR" \
             -DCMAKE_INSTALL_PREFIX:STRING="${PWD}/opt/vast" \
             -DCPACK_PACKAGE_FILE_NAME:STRING="$PACKAGE_NAME" \
-            -DCPACK_GENERATOR:STRING=TGZ \
+            -DCPACK_GENERATOR:STRING=TGZ;DEB \
```

Comment:
The additional Deb should also be uploaded for to GitHub as an artifact and for releases for the `build-debian`  job.

Reply:
We've decided to instead explicitly disable DEB + non-static builds.